docs/docs/architecture.md

```diff
@@ -170,59 +175,62 @@ sequenceDiagram
 
-Inside the enclave `System.out` and `System.err` are wired up to the host console, but there's no filesystem access.
-In release mode any output to `stdout` and `stderr` is suppressed inside the enclave so you can log to `stdout` and 
-`stderr` without worrying that it may leak information outside the enclave when in production.
+#### Adding logs to enclaves
 
-!!! notice
+To add logs to an enclave, you can use:
 
-    In order to provide enclave logging, you have a couple of options.
+1. The [in-memory filesystem](persistence.md) to retain your logs in memory and then export them occasionally to a 
+   trusted client or another enclave.
 
-    You can use the [in-memory  filesystem](persistence.md) to have your logs in-memory and then export them from time to time to a 
-    trusted client or another enclave.
+2. The [persisted filesystem](persistence.md) to persist your logs. When you want to analyze the logs, you need to 
+   send the logs from the enclave to a trusted client.
 
-    Or you can use the [persisted filesystem](persistence.md) to persist your logs. When you want to analyse them, you
-    will need to send the logs from the enclave to a trusted client.
+Alternatively, in sensitive applications, you can decide that nobody should receive the logs when in release mode. 
 
-    Alternatively you may decide that due to the sensitive nature of what your 
-    enclave does, nobody should receive its logs when in release mode: this is a perfectly normal and reasonable 
-    choice! 
+Inside the enclave, `System.out` and `System.err` are wired to the host console, but there's no access to the
+filesystem. In release mode, Conclave suppresses any output to `stdout` and `stderr` inside the enclave. So you can
+add logs to `stdout` and `stderr` without worrying that it may leak production information outside the enclave.
 
 ### Deployment
 
-You can get SGX capable virtual machines from cloud providers. Currently, Microsoft Azure provides the easiest, 
-up-to-date source of SGX VMs. A Conclave host app includes the enclave bundled into the same JAR and the native libraries
-required for working with the kernel driver and Intel infrastructure, so you can deploy a Conclave app by simply using
-regular Java deployment procedures. For example, use the Gradle `assemble` plugin to create a tarball/zip of your
-app, or create a fat JAR and copy it to the server.
-
+You can get SGX-capable virtual machines from cloud providers. Currently, Microsoft Azure provides the easiest, 
+up-to-date source of SGX VMs.
 
+A Conclave host application includes the enclave bundled into the same JAR and the native libraries required for 
+working with the kernel driver and Intel infrastructure. So, you can deploy a Conclave application by using regular
+Java deployment procedures. For example, use Gradle's `assemble` plugin to create a tarball/zip of your application or
+create and copy a fat JAR to the server.
```

Comment:
or create and copy a fat JAR to the server -> `or create a fat jar using the [shadow plugin](https://imperceptiblethoughts.com/shadow/)`.

Reply:
Done.